cypress/support/commands.js

```diff
@@ -101,6 +101,26 @@ Cypress.Commands.add("passCategorization", () => {
     cy.nextPageByButton();
 });
 
+// Go through annotation page, saving default age annotation
+Cypress.Commands.add("passAnnotation", () => {
```

Comment:
`passAnnotation` remains in use for UI-based (not programmatic state-based) testing.